framework/ui/view/iconcodes.h

```diff
@@ -187,6 +187,7 @@ class IconCode
         CHORD_SYMBOL = 0xF352,
         GUITAR_BEND = 0xF353,
         GUITAR_TREMOLO_BAR = 0xF355,
+        MULTIMEASURE_REST = 0xF355,
```

Comment:
2 icons with the same address?

Reply:
0xF355 is the icon code for multimeasure rest. I don’t know where you got the idea that it was the icon code for guitar tremolo bar.

Reply:
I checked in `MusescoreIcon.ttf` and this is certainly the location of the multimeasure rest icon. There doesn't seem to be a tremolo bar icon at all. For this reason, I'll remove the line `GUITAR_TREMOLO_BAR = 0xF355,` and it can be put back if/when an icon is added.

Reply:
@IsaacWeiss @vpereverzev

Can confirm, I've not yet added a guitar tremolo icon.

Reply:
Just updated the icon set:
https://www.dropbox.com/s/ip59ren10u69hr7/MusescoreIcon.ttf?dl=0

Guitar Tremolo is now there: 0xF35C

Reply:
@Tantacrul all taken care of now.